### DISCUSSION TITLE
User Disabled message Due keycloak Implementation

### DISCUSSION THREAD

**Discussion by AnnieMungai** [opensrp/opensrp-client-path-zeir#146] [General]
- [ ] "account has been disabled" error still popping up even if the user has not been disabled. 

Detailed: When a user is logged onto the app and they are inactive for some time or even in the middle of providing services - after being on the tablet for some time, the user is kicked out of the system and gets an error message that the user has been disabled and they should reach the system administrator. 

Questions: 
- Is there a way to disable this message especially when the tablet is still being used? 
- Is there a way to change this message to read something else as it is confusing to the user to have 'user disabled' message? 
- This error has been observed in all applications using keycloak but ZEIR is the only one in production and requires urgent attention. 

Any ideas guys? 
cc @tychus6 @ellykits @ndegwamartin @qiarie

**Comment by ndegwamartin** [comment]
@AnnieMungai Which other deployment in production is experiencing this?

**Reply by AnnieMungai** [reply]
@ndegwamartin Tuninisia staging. 
Is MC still experiencing it? I used to get notifications when it was on my device.
